### ISSUE TITLE
CocoaPods could not find compatible versions for pod "rn-ocr-lib"

### ISSUE THREAD

**Issue by naveenvignesh5** [naveenvignesh5/rn-ocr-lib#1]
<img width="1203" alt="Screenshot 2023-05-20 at 9 32 34 PM" src="https://github.com/naveenvignesh5/rn-ocr-lib/assets/19146999/e981ad72-0c5f-4128-b5af-712987e2130a">

**Comment by naveenvignesh5** [comment]
The project requires minimum deployment target for iOS. Will be updated in README.md. Kindly make the below change in `ios/PodFile` in your react native project.

```ruby
# platform :ios, min_ios_version_supported -- to be removed
platform :ios, '13.0'
```

**Comment by naveenvignesh5** [comment]
Issue resolved in release `v0.2.3`
